geomesa-cassandra/geomesa-cassandra-datastore/src/main/scala/org/locationtech/geomesa/cassandra/index/AttributeColumnMapper.scala

```diff
@@ -59,18 +62,18 @@ class AttributeColumnMapper(shards: Seq[ColumnSelect]) extends CassandraColumnMa
 
   override def select(range: ScanRange[_], tieredKeyRanges: Seq[ByteRange]): Seq[RowSelect] = {
     val primary = range.asInstanceOf[ScanRange[AttributeIndexKey]] match {
-      case SingleRowRange(row)   => Seq(ColumnSelect(Value, row.value, row.value))
-      case BoundedRange(lo, hi)  => Seq(ColumnSelect(Value, lo.value, hi.value))
-      case LowerBoundedRange(lo) => Seq(ColumnSelect(Value, lo.value, null))
-      case UpperBoundedRange(hi) => Seq(ColumnSelect(Value, null, hi.value))
-      case PrefixRange(prefix)   => Seq(ColumnSelect(Value, prefix.value, prefix.value + "zzzz")) // TODO ?
+      case SingleRowRange(row)   => Seq(ColumnSelect(Value, row.value, row.value, startInclusive = true, endInclusive = true))
+      case BoundedRange(lo, hi)  => Seq(ColumnSelect(Value, lo.value, hi.value, lo.inclusive, hi.inclusive))
+      case LowerBoundedRange(lo) => Seq(ColumnSelect(Value, lo.value, null, lo.inclusive, endInclusive = false))
+      case UpperBoundedRange(hi) => Seq(ColumnSelect(Value, null, hi.value, startInclusive = false, hi.inclusive))
+      case PrefixRange(prefix)   => Seq(ColumnSelect(Value, prefix.value, prefix.value + "zzzz", prefix.inclusive, endInclusive = false)) // TODO ?
```

Comment:
Is the TODO still applicable?  And if so, it is worth a ticket?

Reply:
it is still applicable, but I think if we create a ticket it will never get looked at, so...

Reply:
it seems to work well enough the way it is, i guess